src/.vuepress/public/editor.html

```diff
@@ -281,6 +281,10 @@ <h5>Future features</h5>
           <p><input id="option-feature-referencetypes" type="checkbox" /><label for="option-feature-referencetypes"> Enable Reference Types</label></p>
           <p><input id="option-feature-wasm-gc" type="checkbox" /><label for="option-feature-wasm-gc"> Enable WebAssembly GC</label></p>
         </div>
+        <h5>Text format</h5>
+        <div class="option">
+          <p><input id="option-use-wast" type="checkbox" /><label for="option-use-wast"> Use stack text format (wast)</label></p>
```

Comment:
```suggestion
          <p><input id="option-use-wast" type="checkbox" /><label for="option-use-wast"> Use non-standard S-expression text format (wast)</label></p>
```
Non-standard because, in Binaryen, it includes at least the `pop` instruction (critical to consume the thrown value in exception-handling) that is not part of the standard. I've been thinking for a while that we should probably propose it to Wasm to fix the asymmetry, but that's a different story :)